app/controllers/posts_controller.rb

```diff
@@ -3,11 +3,14 @@ class PostsController < ApplicationController
 
   # GET /posts or /posts.json
   def index
-    @posts = Post.all
+    @user = User.find(params[:id])
+    @posts = @user.posts
   end
 
   # GET /posts/1 or /posts/1.json
-  def show; end
+  def show
+    @post = Post.find(params[:post_id])
+  end
 
   # GET /posts/new
   def new
```

Comment:
As part of the project requiremens you were asked NOT to use `rails generate scaffold` command to generate the project/controllers since this project is to show you how to set up the controllers manually by yourself. However, I see that you have used the `rails generate scaffold`, hence a lot of code in the controller which are not needed at this milestone. Kindly consider removing all the boilerplate code generated by the `rails generate scaffold` and consider creating the controllers manually. 
The posts_controller is suppose to have the #index and #show actions only for now.